.golangci.yml

```diff
@@ -21,6 +21,15 @@ linters-settings:
       - '^log\.'
       - '^print$'
       - '^println$'
+      # Use private/pkg/protoencoding Marshalers and Unmarshalers
+      - '^proto.Marshal$'
+      - '^proto.Unmarshal$'
+      - '^proto.MarshalOptions$'
+      - '^proto.UnmarshalOptions$'
+      - '^protojson.Marshal$'
+      - '^protojson.Unmarshal$'
+      - '^protojson.MarshalOptions$'
+      - '^protojson.UnmarshalOptions$'
```

Comment:
Should this include YAML, too?

Reply:
Ugh probably - @emcfarlane any chance you can commandeer and add?